src/content/docs/datahub/datahub-files.mdx

```diff
@@ -1,41 +1,59 @@
 ---
-title: Working with files
+title: How to work with files in the DataHUB
```

Comment:
Maybe rename to "Manage files", this is shown in the sidebar under "DataHUB" and we should keep titles short and descriptive